src/style.css

```diff
@@ -2,2 +1,103 @@
+body,
+h1,
+ul,
+li,
+p,
+a {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  background-color: #f9f9f9;
+  width: 100%;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.main-header {
+  width: 100%;
+  max-width: 1200px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+  align-items: center;
+  background-color: #fff;
+  position: fixed;
+  font-size: 12px;
+  margin: 0 auto;
+  box-sizing: border-box;
+  font-weight: 500;
+}
+
+.logo img {
+  height: 40px;
+  width: auto;
+  display: block;
+}
+
+.nav-menu {
+  list-style: none;
```

Comment:
Don't use tag names for styling, style elements using `.class` instead, except for `html` and `body`. For example, instead of `ul` use `.nav-menu` to set the list-style.